src/styles.scss

```diff
@@ -1 +1,5 @@
 /* You can add global styles to this file, and also import other style files */
+@import 'scss/variables.scss';
```

Comment:
you should use `@use` syntax and in the file you will use it like:
```
.element {
    color: variables.$green-color
}
```

You put `variables.` as it's the file name